include/RMGGeneratorDecay0.hh

```diff
@@ -41,12 +79,59 @@ class RMGGeneratorDecay0 : public RMGVGenerator {
     RMGGeneratorDecay0(RMGGeneratorDecay0&&) = delete;
     RMGGeneratorDecay0& operator=(RMGGeneratorDecay0&&) = delete;
 
+    /** @brief Calls to @c fDecay0G4Generator to generate the primary for the event.
+     */
     void GeneratePrimaries(G4Event*) override;
     inline void SetParticlePosition(G4ThreeVector) override{};
 
+    /** @brief Updates the seed and configuration at the beginning of each run.
+     *  @details Only does something if any remage set up command was used ( @c fUpdateSeeds is true).
+     */
+    void BeginOfRunAction(const G4Run*) override;
+    inline void EndOfRunAction(const G4Run*) override {}
+
+    /** @brief Sets BxDecay0 to run in background mode and sets the specific isotope.
+     *  @param isotope The isotope to set (e.g. "Co60").
+     */
+    void SetBackground(std::string);
+
+    void SetUpdateSeeds(bool value) { fUpdateSeeds = value; }
```

Comment:
do we need this function. This is only needed for resetting this flag from external code, but do we want this?